### PR TITLE
[hotfix]multi_account_add_email

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
fix https://trello.com/c/iuQqlj9Z/88-making-email-primary-while-logged-in-as-another-user-sends-confirmation-email-to-an-already-confirmed-user.

### CODE REVIEW

**Comment by chennan47** [comment]
@sloria rdy for review

File: website/profile/views.py [hunk 1]
```diff
@@ -95,18 +95,25 @@ def update_user(auth):
...
         ]
 
         if user.username in removed_emails:
             raise HTTPError(httplib.FORBIDDEN)
 
         for address in removed_emails:
+            if address == user.username:
+                raise HTTPError(httplib.FORBIDDEN)
             if address in user.emails:
                 user.remove_email(address)
```
Comment (sloria) [inline website/profile/views.py]: As a safeguard, it would be good to have `User#remove_email` raise an error if passed the user's username.

```python
try:
    user.remove_email(address)
except InvalidActionError:
    raise HTTPError(400, ....'Cannot remove primary email')
```

Though this can be in a separate PR.

Comment (sloria) [inline website/profile/views.py]: Same with `User#remove_unconfirmed_email`; you shouldn't be able to pass an email address that isn't in the user's unconfirmed emails list.
